tools/cli.js

```diff
@@ -116,9 +124,41 @@ commandProcessor.complete = (inputs, depth) => {
 
 commandProcessor.call = (interfaceName, methodName, args, callback) => {
   if (!state.client) return callback(new Error('Not connected'));
+
+  interfaceName = tryComplete(interfaceName, Object.keys(state.api));
```

Comment:
@lundibundi I'd say I'm -1 on it one way or another, but I will not block the PR if I am in the minority.